flocker/provision/_install.py

```diff
@@ -255,6 +255,20 @@ def install_commands_yum(package_name, distribution, package_source,
             'cp', '/tmp/clusterhq-build.repo',
             '/etc/yum.repos.d/clusterhq-build.repo']))
         repo_options = ['--enablerepo=clusterhq-build']
+
+        # There might be a clusterhq-build repo already and its metadata will
+        # be cached.  Wipe out the cache so that metadata gets loaded from the
+        # latest build URL.
+        # XXX: Would it be better to do this during uninstallation?
+        commands.append(
+            run_from_args([
+                b"yum",
+                b"--disablerepo='*'",
```

Comment:
Since the clusterhq-build repo isn't enabled by default, we need to enable it explicitly in order to clean the data from it.